Greatest number/GreatestNumber.java

```diff
@@ -0,0 +1,37 @@
+//take three numbers from user and find the greatest number
+
+import java.util.Scanner;
+
+public class GreatestNumber
+  {
+    public static void main(String[] args)
+      {
+	Scanner input = new Scanner(System.in);
+	System.out.println("Enter first number : ");
+	int num1 = input.nextInt();  //user enters first number
+	System.out.println("Enter second number : ");
+	int num2 = input.nextInt();  //user enters second number
+	System.out.println("Enter third number : ");
+	int num3 = input.nextInt();  //user enters third number
+	
+	if(num1>num2)
```

Comment:
This solution looks good. 

But we can also do it in a better way by maintaining an array of integer values. 
E.g: `int mums[] = new int[N];`
You can obtain 'N' value in the beginning and ask for numbers till 'N', and store each input in the array. Finally, you can use `for()` loop to find the greatest number.